doc/snmp/snmp-changes-to-support-ipv6.md

```diff
@@ -0,0 +1,138 @@
+# SONiC SNMP Changes to support IPv6 #
+
+This document captures the changes required to support SNMP over IPv6.
+
+## Motivation ##
+
+SNMP query over IPv6 address fails in certain scenarios on single asic platforms.
+Ideally, SNMP over IPv6 should be successful over both IPv4 and IPv6.
```

Comment:
Could you rephase "SNMP over **IPv6** should be successful over both **IPv4** and IPv6"?